### PR TITLE
feat: Vaccine lookup

### PR DESCRIPTION
**Pull request by SveinungFjaer**

### CODE REVIEW

File: components/registration/default-registration-form.html [hunk 1]
```diff
@@ -6,15 +6,23 @@
...
             </td>
             <td>
-                <button  ng-click="showLabTest()" class="btn btn-secondary"  ng-disabled="!shouldShowLabTest()">Vis prøvesvar</button>
+                <button ng-click="showVaccine()" type="button" class="btn btn-secondary" ng-disabled="!shouldShowLabTestAndVaccine()">Vis vaksinestatus</button>
             </td>
         </tr>
     </table>
-    <div ng-show="!shouldShowLabTest()" class="alert alert-warning" style="font-size: 9pt;">Gyldig nasjonalt felles hjelpenummer eller F-nummer må være satt for å vise prøvesvar</div>
+    <div ng-show="!shouldShowLabTestAndVaccine()" class="alert alert-warning" style="font-size: 9pt;">Gyldig nasjonalt felles hjelpenummer eller F-nummer må være satt for å vise prøvesvar og vaksinestatus</div>
+    <hr ng-if="selectedProgram.id == 'uYjxkTbwRNf' && selectedEnrollment.enrollment" />
```
Comment (Bekkalizer) [inline components/registration/default-registration-form.html:25]: What is this line for - and why is it showing for this program? Should it rather use shouldShowLabTestAndVaccine()?

File: components/registration/registration-controller.js [hunk 2]
```diff
@@ -1422,31 +1422,31 @@ trackerCapture.controller('RegistrationController',
         return attributeMandatory || $scope.mandatoryFields[attributeId];
     }
 
-    $scope.labTestLookupNr = undefined;
-    $scope.labTestLookupNrGetterSetter = function(value) {
+    $scope.fNrOrEquivalent = undefined;
+    $scope.fNrOrEquivalentGetterSetter = function(value) {
```
Comment (Bekkalizer) [inline components/registration/registration-controller.js:1426]: Do we ever need to "set" this? Can we just as well use the scope.getBestNumberForLabTestAndVaccine directly?

File: components/registration/default-registration-form.html (hunk 1 above)
Comment (SveinungFjaer) [reply components/registration/default-registration-form.html:25]: The line is used to separate prøvesvar and vaksine from klinikermelding. (see line 26). It felt like it was needed to not get the buttons to cluttered. But open for suggestions for other ways to clean things up

image

Comment (Bekkalizer) [reply components/registration/default-registration-form.html:25]: Ah got it! It is only needed for the covid case program, as that's the only place you have both the lookups and the button for klinikermelding.

**Pull request merged**